docs/Modding/Rulesets/effects.rst

```diff
@@ -809,6 +809,14 @@ Has_Senate
 
 Inspire_Partisans
     Partisan units (defined in :file:`units.ruleset`) may spring up when this player's cities are taken.
+    The number of partisans created is a function of the city size.
+    When multiple unit types have the Partisan flag, the game tries to choose one the player can build. If
+    the player cannot build any, the first Partisan unit type is used. If the created partisans units have a
+    non-zero ``pop_cost``, the city will lose citizens accordingly.
+
+    Note that this effect can create units that the player could normally not build (because the tech hasn't
```

Comment:
Make this an RST note `.. note::`?

Reply:
Updated:
![image](https://github.com/longturn/freeciv21/assets/22327575/3cdb841c-4510-4e6b-a78d-4fcb197d6fc4)